frontend/kesaseteli/employer/src/components/footer/Footer.tsx

```diff
@@ -0,0 +1,18 @@
+import { useTranslation } from 'next-i18next';
+import React from 'react';
+
+import { Footer } from './Footer.sc';
+
+const FooterSection: React.FC = () => {
+  const { t } = useTranslation();
+  return (
+    <Footer title={t('common:appName')}>
```

Comment:
I have a question regarding the naming, how do you distinguish between StyledComponents and other "normal" components. 
In Benefits, all Styled Components are prefixed with `Styled`, which is IMO makes the components' names too long. However, there is another approach I know of, which goes like this:
```tsx
import * as SC from './Footer.sc';
...
return (<SC.Footer title={t('common:appName')}>
...
```
This way it will be easier to spot Styled Components. 
It would be good if we agree on one convention and follow it on both of the projects.

Reply:
I also commented previously that the `<Styled*>` is too boiler-plate and makes harder to read. I think that the `SC.` is good prefix, I can refactor this to use it.